### DISCUSSION TITLE
Large disparity between provider data

### DISCUSSION THREAD

**Discussion by s38b35M5** [Akylas/oss-weather#119] [General]
In my use of the app, I am finding it hard to trust the forecast and even current data of any one provider.

At the current time, the three providers show current temperatures of 79F, 79F, and 86F. It is truly 74F here. Meteo-France animated graphic always says it's currently raining when it has not rained in almost a month. Open-Meteo always shows apocalyptic current and hi/low temperatures (it currently says it's 86F here and a high of 97) that are more than 10F higher than actual. 

Comparing to a closed source app and provider (Wundergound) whose data I trust (but whose motives I do not), none of the providers in this app are as close to actual conditions in forecast temps or weather events.

This disparity and wide variability without apparent cause leads me to distrust the app in general.

To replicate my findings:

1. Search for Placencia Belize and add location
2. Swipe on location name to switch between providers
3. Compare to https://www.wunderground.com/weather/bz/placencia

**Comment by 0-5-0** [comment]
Having tried many apps it's basically always a guessing game between providers, however, with OSS Weather the Open-Meteo models are exposed and can be chosen under the preference menu.

You could try different Open-Meteo weather models (ECMWF, DWD ICON, MeteoFrance, ARPAE, etc.) to see if one works better than another. I've found "Best Match" in my case might not always pick the best match (I wanted GEM and got GFS for example).

I believe the seamless models means it'll combine their short range forecasts and long range forecast models into one view/one list so you can still view forecasts 14 days out from the global models while still getting ultra accurate 2-3 day forecasts from the same provider's regional model. 

Maybe there's an Open-Meteo model for you closer to what WeatherUnderground provides if WU is generally more reliable for your area. If you do go through and find one let us know, it might be useful for others :).

**Reply by s38b35M5** [reply]
I'll play some more. I really want to use this app as my primary, but need to get the data closer to reality first

**Reply by 0-5-0** [reply]
It can be frustrating some times lol, hopefully there's a model that works well for you. I'm certainly glad we can at least have the choice to pick the model, it's the only Open-Meteo app that does it as far as I know.

One thing that's in the plans but which will take time to add is having temperature and precipitation forecasts from the different models all stacked together in the chart at the top, so you could basically see how many models agree with each other and make a better decisions that way.

The developer had shared this as a loose example: https://www.meteoblue.com/fr/meteo/prevision/multimodel/paris_france_2988507

**Comment by farfromrefug** [comment]
@s38b35M5 yes it can really demands on location/provider. 
there is always the possibility of adding new providers. but we need to find one that works for you.
you could try breezy weather which has a lot of other providers. if one works for you and you want me to add it I will
